components/slider/w-slider.vue

```diff
@@ -25,6 +29,7 @@ const props = defineProps({
   label: String,
   labelledBy: String,
   disabled: Boolean,
+  preventAcceleration: Boolean,
```

Comment:
Is this used anywhere?

Reply:
In line 44 in the examples (Slider.vue) 😄  ...it is just added to be able to opt-out of the acceleration if you for some reason don't want it.